core-bundle/src/Resources/contao/config/default.php

```diff
@@ -13,7 +13,9 @@
 $GLOBALS['TL_CONFIG']['adminEmail']     = '';
 $GLOBALS['TL_CONFIG']['enableSearch']   = true;
 $GLOBALS['TL_CONFIG']['indexProtected'] = false;
-$GLOBALS['TL_CONFIG']['folderUrl']      = false;
+
+/** @deprecated Deprecated since Contao 4.9, check tl_page.useFolderUrl instead. */
```

Comment:
This is not necessary. There are plenty of other deprecated keys in this file as well.